package.json

```diff
@@ -8,7 +8,8 @@
     "main": "dist/oneui.min.js",
     "scripts": {
         "storybook": "start-storybook -s ./stories/static -p 9001 -c .storybook",
-        "storybook:prod": "build-storybook -s ./stories/static -c .storybook",
+        "storybook:build": "build-storybook -s ./stories/static -c .storybook",
+        "storybook:deploy": "storybook-to-ghpages --script=storybook:build --out=.out",
```

Comment:
Now we have two generated folders for the storybook static files.: `.out` created by `storybook:deploy`, and `storybook-static` created by the `storybook:build` command. Perhaps make sense to configure them to output in the same folder

Reply:
Where do you see `storybook-static`? If you mean `./stories/static`, that's the directory where to load static files from (e.g. avatar image)

Reply:
If you run `storybook:build` you can see

Reply:
I just checked and when running `storybook:deploy` I don't see any `storybook-static` directory. Maybe it's a leftover from another branch?

Reply:
Run only `storybook:build`. Doesn't it create the static files in the `storybook-static` folder ?

Reply:
Done